### ISSUE TITLE
Warning when pausing unfinished cybran naval factory

### ISSUE THREAD

**Issue by speed2CZ** [FAForever/fa#1131]
```
WARNING: Error running OnPaused script in Entity urb0103 at 1ba0d608: ...\gamedata\units.scd\units\urb0103\urb0103_script.lua(37): attempt to call method `SetGoal' (a nil value)
         stack traceback:
         	...\gamedata\units.scd\units\urb0103\urb0103_script.lua(37): in function `StopArmsMoving'
         	...s\multimedia\documents\github\fa\lua\cybranunits.lua(253): in function <...s\multimedia\documents\github\fa\lua\cybranunits.lua:251>
```
In current beta

**Comment by aeoncleanse** [comment]
I think this is one of the logspam errors I tried to get rid of before (I failed). It didn't crash the sim right?

**Comment by speed2CZ** [comment]
no it didn't

**Comment by aeoncleanse** [comment]
Verified. This will be the same as some of the other stuff fixed in dee8d98c4934e263d6b9e47fcaf3818711302e1a

**Comment by aeoncleanse** [comment]
d533d6c0b50bd1eb0677969b10a400ca248c60f5
